### ISSUE TITLE
Option to create a dialog.

### ISSUE THREAD

**Issue by WopsS** [WopsS/PawnPlus#13]
An option to create a dialog with preview directly in PawnPlus, without need to test it in SA-MP.

**Comment by Malganys** [comment]
Yeah, this would be nice. But add multiple options to create, like default dialog native, y_dialog and easyDialog.
